### DISCUSSION TITLE
Looking for Resources to Start Developing QField Plugins

### DISCUSSION THREAD

**Discussion by paul-carteron** [opengisch/QField#5867] [Q&A]
Hi QField Community,

I'm interested in developing plugins for QField but have been struggling to find tutorials or resources to get started. The most helpful resource I've found so far is the article [Supercharge your fieldwork with QField’s project and app-wide plugins](https://www.opengis.ch/fr/2024/06/18/supercharge-your-fieldwork-with-qfields-project-and-app-wide-plugins/).

Do you have any recommendations for learning materials or a roadmap to guide QField plugin development? Any advice or pointers would be greatly appreciated!

Thank you in advance!

**Comment by nirvn** [comment]
@paul-carteron , have you stumbled on this github topics page which regroups a growing number of plugins: https://github.com/topics/qfield-plugin ? They are all offering source code for you to learn from.

This webinar might also be a useful resource: https://drive.google.com/file/d/1s5SD-JpCq9QHoIbmOMEqDsTyGSsN6QrC/view 

(We'd also most welcome a bit of an intro how-to if you are interested in writing such a document once you have learned enough.)

**Reply by nirvn** [reply]
@paul-carteron , actually, this webinar might also be a useful resource: https://drive.google.com/file/d/1s5SD-JpCq9QHoIbmOMEqDsTyGSsN6QrC/view

Keep us posted on your progress :)

**Comment by paul-carteron** [comment]
@nirvn I am currently exploring [https://github.com/topics/qfield-plugin](https://github.com/topics/qfield-plugin) ;) I’ll also check out the webinar. Thank you for your help!
